### PR TITLE
Edited the ReadMe.

### PR DESCRIPTION
**Pull request by majortom64**
Added one space and an example for using the `CognitoBasicAuthenticator`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,21 @@ struct User: Content & Authenticatable {
     }
 }
 ```
+
+An example using `CognitoBasicAuthenticator`:
```
Comment (adam-fowler) [inline README.md:69]: Given there are two examples perhaps we should give them sub-headings. 
```
### Basic Authenticator Example
```

File: README.md [hunk 2]
```diff
@@ -65,6 +65,21 @@ struct User: Content & Authenticatable {
...
+
+```swift
+        let cognitoBasicAuth = routes.grouped(CognitoBasicAuthenticator())
+        cognitoBasicAuth.post("login", use: login)
```
Comment (adam-fowler) [inline README.md:73]: You need to add guard middleware to get the 401 response.
```
let cognitoBasicAuth = app.grouped(CognitoBasicAuthenticator())
    .grouped(User.guardMiddleware())
cognitoBasicAuth.post("login", use: login)
```

File: README.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ func authenticateAccess(_ req: Request) throws -> Future<> {
...
+Three authenticators are available. See the [Vapor docs](https://docs.vapor.codes/4.0/authentication) for more details on authentication in Vapor. `CognitoBasicAuthenticator` will do username, password authentication and returns a `CognitoAuthenticateResponse`. `CognitoAccessAuthenticator` will do access token authentication and returns an `CognitoAccessToken` which holds all the information that could be extracted from the access token. `CognitoIdAuthenticator<Payload>` does id token authentication and extracts information from the id token into your own `Payload` type. The standard list of claims that can be found in an id token are detailed in the [OpenID spec] (https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims). Your `Payload` type needs to decode using these tags, the username tag "cognito:username" and any custom tags you may have setup for the user pool. Below is an example of using the id token authenticator.
```
Comment (adam-fowler) [inline README.md:55]: You placed your basic authenticator example in the middle of my Id token example. Given the user is going to first implement the basic authenticator maybe place it above. See below about sub-headings. You'll need to remove the "Below is an example..." text from above

**Review by adam-fowler — CHANGES_REQUESTED** [review]

**Comment by adam-fowler** [comment]
Hey @majortom64 do you intend to update this?

File: README.md (hunk 3 above)
Comment (majortom64) [reply README.md:55]: Now that I have pretty much finished my example, I  will add some other examples and generally clean up my text. I should be have it done with in a day or two.

**Pull request closed without merging**